account_payment_unece/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Account Payment UNECE
+=====================
+
+This module adds a field *UNECE Payment Mean* on *Payment Methods* to be able to use a standard written by the `United Nations Economic Commission for Europe <http://www.unece.org>`_ (which has 56 members states in Europe, America and Central Asia, cf `Wikipedia <https://en.wikipedia.org/wiki/United_Nations_Economic_Commission_for_Europe>`_). This standard define a codification of the payment modes, cf the `official code list <http://www.unece.org/trade/untdid/d99b/tred/tred4461.htm>`_.
+
+This codification is used for example in the two main international standards for electronic invoicing:
+
+* `Cross Industry Invoice <http://tfig.unece.org/contents/cross-industry-invoice-cii.htm>`_ (CII),
+* `Universal Business Language <http://ubl.xml.org/>`_ (UBL)
+
+Configuration
+=============
+
+Go to the menu *Accounting > Configuration > Management > Payment Methods* and select the appropriate value for the *UNECE Code*.
```

Comment:
True. You need the module account_payment_mode to see this menu entry, but we don't depend on this module.